src/roller.js

```diff
@@ -100,7 +100,7 @@ class LMRTFYRoller extends Application {
     _makeRoll(event, rollMethod, ...args) {
         let fakeEvent = {}
         if (this.advantage === 0) {
-            fakeEvent.shiftKey = true;
+            fakeEvent.shiftKey = !LMRTFY.defaultQuery;
```

Comment:
I'm not sure about the name of this variable `defaultQuery` doesn't make a lot of sense to me, but it's not a big issue.
Also noticing, it should be set maybe for the advantage/disadvanage sections?  I'm not sure.
Actually, what if the system uses Shift for advantage instead of query.. if every system decides on their own modifiers, then it would make more sense to have `ShiftNormal`, `ShiftAdvantage`, `ShiftQuery`, etc.. and `CtrlNormal`, etc... 
Not great, but 🤷 what do you think ?

Reply:
When you pass the shift key in PF2e it sets it to 'query player for mods/adv/disadv' automatically. In 5e the **default** is to **query** the player, so shift key is true. It's a bit of a fast/loose with it and the inverse down on 114. 

The problem in the definitions you make is that **ShiftNormal** means different things for the different systems, I could use it in this manner but I personally think it only makes sense depending on what you think normal behavior for a shift button is true. I was trying to stay out of that and be neutral - but I agree the name isn't great; ¯\_(ツ)_/¯ Same situation -- can always modify it?

Reply:
Maybe you misunderstood what I meant.
Does this help make it clearer ? 
```js
if (pf2) {
  LMRTFY.ShiftNormal = false;
  LMRTFY.AltNormal = false;
  LMRTFY.CtrlNormal = false;

  LMRTFY.ShiftAdvantage = false;
  LMRTFY.AltAdvantage = false;
  LMRTFY.CtrlAdvantage = true;

  LMRTFY.ShiftDisadvantage = false;
  LMRTFY.AltDisadvantage = true;
  LMRTFY.CtrlDisadvantage = false;

  LMRTFY.ShiftQuery = true;
  LMRTFY.AltQuery = false;
  LMRTFY.CtrlQuery = false;
} else {
  LMRTFY.ShiftNormal = true;
  LMRTFY.AltNormal = false;
  LMRTFY.CtrlNormal = false;

  LMRTFY.ShiftAdvantage = false;
  LMRTFY.AltAdvantage = false;
  LMRTFY.CtrlAdvantage = true;

  LMRTFY.ShiftDisadvantage = false;
  LMRTFY.AltDisadvantage = true;
  LMRTFY.CtrlDisadvantage = false;

  LMRTFY.ShiftQuery = false;
  LMRTFY.AltQuery = false;
  LMRTFY.CtrlQuery = false;
}
```
And then in the call itself, do something like this : 
```js
   let type = "Normal";
   if (advantage == 1) type = "Advantage";
   else if (advantage == -1) type = "Disadvantage";
   else if (advantage == 2) type = "Query";

    fakeEvent.shiftKey = LMRTFY[`Shift${type}`];
    fakeEvent.altKey = LMRTFY[`Alt${type}`];
    fakeEvent.ctrlKey = LMRTFY[`Ctrl${type}`];
```

Reply:
In d20Roll this is a fastForward Normal roll; which is why it wouldn't also be required in the Advantage/Disadvantage sections:

```typescript
   * Holding SHIFT, ALT, or CTRL when the attack is rolled will "fast-forward".
   * This chooses the default options of a normal attack with no bonus, Critical, or no bonus respectively
```
So perhaps just using that terminology? Seems awkward and verbose still.. we'd want to specify "fast forward default behavior for normal/advantage/disadvantage"

Reply:
I see, but I think if you look at it from what you are ultimately achieving with the fake events is that you are doing a FastForward roll of either Normal Roll, Advantage Roll, or Disadvantage Roll. 

```
* Holding SHIFT, ALT, or CTRL when the attack is rolled will "fast-forward".
* This chooses the default options of a normal attack with no bonus, Advantage, or Disadvantage respectively
```

So you really are only worried about the default behavior of Shift -> base roll, Alt -> Adv. roll, Ctrl -> Disadv. Roll -- the other two key events should always be false in those scenarios. It seems like a lot of extra data that is non-essential, but it does make the key assignment easier. I'm fine with adding it in like this, just wanted to add my two cents here.

Reply:
I made some adjustments to the idea and essentially assigned the fake key event per system.